docs/src/02-metadata-merging.jmd

```diff
@@ -186,10 +191,13 @@ filter!(r-> !any(ismissing, [r[:studyID], r[:sampleID], r[:timepoint], r[:metada
 disallowmissing!(allmeta)
 
 # show a random assortment of ~ 10 rows
+# **shows random assortment of 4 rows x 6 columns**
+# **What does it mean when `sampleID`s end in `M`. Not clear what that's meant to mean.**
```

Comment:
Can you screenshot this for me? I'm not seeing any sample ID's ending in `M`. I had to generate a new version that doesn't show that (but of course shows a different subset of rows... maybe I should fix that so it's consistent), and I looked in the parent tables and don't see it. 

We should track it down though, I'm worried it's supposed to indicate a maternal brain scan, but my code will assume it's timepoint 13.

Reply:
<img width="893" alt="Screen Shot 2019-07-08 at 10 51 59 AM" src="https://user-images.githubusercontent.com/30243213/60819949-769d3700-a16e-11e9-900d-39a6b4c72719.png">

Reply:
Ahh, I see. That's [from here](https://github.com/Klepac-Ceraj-Lab/echo_analysis/blob/master/docs/src/02-metadata-merging.jmd#L154-L176)